packages/shorebird_cli/lib/src/executables/patch_executable.dart

```diff
@@ -49,9 +50,20 @@ class PatchExecutable {
     final result = await process.run(diffExecutable, diffArguments);
 
     if (result.exitCode != ExitCode.success.code) {
+      var messageDetails = '';
+
+      if (result.exitCode == -1073741515 && platform.isWindows) {
```

Comment:
I would pull this out into a const with a comment and a link explaining the significance of this value